syntax = "proto3";
package user_admin;




import "google/protobuf/struct.proto";

/*
ListUsersInfo请求
*/
message ListUsersInfoRequest {
    /*
    状态
    */
    string state = 1 ;
    /*
    页码
    */
    int32 page = 2 ;
    /*
    页大小
    */
    int32 pageSize = 3 ;
}

/*
ListUsersInfo返回
*/
message ListUsersInfoResponse {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 pageSize = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    用户列表
    */
    repeated google.protobuf.Struct list = 4 ;
}

/*
ListUsersInfoApi返回
*/
message ListUsersInfoResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    user_admin.ListUsersInfoResponse data = 4 ;
}

